vividus-extension-selenium/src/main/java/org/vividus/ui/action/search/AbstractElementAction.java

```diff
@@ -105,7 +108,13 @@ public LocatorType getType()
                 LOGGER.warn(e.getMessage(), e);
                 return false;
             }
-        }).collect(Collectors.toList());
+        }).collect(Collectors.collectingAndThen(Collectors.toList(), list ->
+        {
+            LOGGER.atInfo().addArgument(visibility)
+                           .addArgument(list::size)
+                           .log("Number of {} elements is equal to {}");
```

Comment:
how does the message look like?

Reply:
Number of VISIBLE elements is equal to 1

Reply:
it should be like this: `Number of visible elements is 1`